### PR TITLE
XWIKI-21375: Videos on the help page don't have alternatives

### PR DESCRIPTION
**Pull request by Sereza7**
## Jira
https://jira.xwiki.org/browse/XWIKI-21375
## PR Changes
* Replaced unstandard aria attribute
## Note
This is a fix for the webstandard regression https://ge.xwiki.org/s/fz2vfc7qvnhue/tests/goal/org.xwiki.platform:xwiki-platform-distribution-flavor-test-webstandards:surefire:test@default/details/org.xwiki.test.webstandards.framework.DefaultValidationTest/Validating%20HTML5%20validity%20for%20%5Bhttp:%2F%2F127.0.0.1:8080%2Fxwiki%2Fbin%2Fview%2FHelp%2FVideos%2FWebHome%5D%20executed%20with%20credentials%20Admin:admin?expanded-stacktrace=WyIwIl0&top-execution=1 introduced in https://github.com/xwiki/xwiki-platform/commit/877d3e64a06ae4814356301f285ec0013178825a

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/Videos/WebHome.xml [hunk 1]
```diff
@@ -136,8 +136,11 @@
 #macro (helpVideoCard $data)
   &lt;div class="videoCard"&gt;
     &lt;iframe src="$data.url" allowfullscreen title="$escapetool.xml($data.title)" 
-        aria-description="$escapetool.xml($services.localization.render('help.videos.hint'))" &gt;
+        aria-describedby="video_hint_$foreach.index" &gt;
     &lt;/iframe&gt;
+    &lt;span id="video_hint_$foreach.index" class="sr-only"&gt;
```
Comment (michitux) [inline xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/Videos/WebHome.xml:141]: Didn't we conclude that texts used by `aria-describedby` should use `hidden` or do you think this should be available independent of the iframe?

**Comment by Sereza7** [comment]
Need to also fix CustomDutchWebValidationGuidelines error:

> junit.framework.AssertionFailedError: 
> Validation errors in xwiki:Help.Videos.WebHome()
> ERROR: The use of role is mandatory in iframes.
> ERROR: The use of role is mandatory in iframes.
> ERROR: The use of role is mandatory in iframes.
> ERROR: The use of role is mandatory in iframes.
> ERROR: The use of role is mandatory in iframes.
> ERROR: The use of role is mandatory in iframes.
> 
> Extracts of the invalid content:
> 
>   1	<!DOCTYPE html>
> 	at junit.framework.Assert.fail(Assert.java:57)
> 	at junit.framework.TestCase.fail(TestCase.java:223)
> 	at org.xwiki.test.webstandards.framework.DefaultValidationTest.testDocumentValidity(DefaultValidationTest.java:223)
> 	at org.xwiki.test.webstandards.CustomDutchWebGuidelinesValidationTest.testDocumentValidity(CustomDutchWebGuidelinesValidationTest.java:52)
> 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
> 	at junit.framework.TestCase.runTest(TestCase.java:177)
> 	at junit.framework.TestCase.runBare(TestCase.java:142)
> 	at junit.framework.TestResult$1.protect(TestResult.java:122)
> 	at junit.framework.TestResult.runProtected(TestResult.java:142)
> 	at junit.framework.TestResult.run(TestResult.java:125)
> 	at junit.framework.TestCase.run(TestCase.java:130)
> 	at junit.framework.TestSuite.runTest(TestSuite.java:241)
> 	at junit.framework.TestSuite.run(TestSuite.java:236)
> 	at junit.framework.TestSuite.runTest(TestSuite.java:241)
> 	at junit.framework.TestSuite.run(TestSuite.java:236)
> 	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)
> 	at junit.extensions.TestSetup$1.protect(TestSetup.java:23)
> 	at junit.framework.TestResult.runProtected(TestResult.java:142)
> 	at junit.extensions.TestSetup.run(TestSetup.java:27)

**Comment by Sereza7** [comment]
Addressed the regression mentioned in the previous message in https://github.com/xwiki/xwiki-platform/pull/2484/commits/74ee2f462a0d1d4e43e371c6526ead972ccfbba0

**Comment by Sereza7** [comment]
Addressed the regression mentioned in the previous message in https://github.com/xwiki/xwiki-platform/pull/2484/commits/74ee2f462a0d1d4e43e371c6526ead972ccfbba0

**Pull request merged**